app/schemas/de.tum.in.tumcampusapp.database.TcaDb/10.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
```

Comment:
This file should not be changed @schulzalexander

Reply:
right, I'll fix that.